country-a-service/integration-tests/src/test/java/dk/sundhedsdatastyrelsen/ncpeh/MinLogIT.java

```diff
@@ -0,0 +1,15 @@
+package dk.sundhedsdatastyrelsen.ncpeh;
+
+import dk.nsp.test.idp.OrganizationIdentities;
+import dk.sundhedsdatastyrelsen.ncpeh.service.MinLogService;
+import dk.sundhedsdatastyrelsen.ncpeh.testing.shared.MinLog;
+import org.junit.jupiter.api.Test;
+
+public class MinLogIT {
+    private final MinLogService service = new MinLogService(MinLog.apiClient());
+
+    @Test
+    void testEvent() {
+        service.logEventOnPatient(MinLog.cprJensJensenReadOnly, "integrationstest", MinLog.maltaHealtcareOfficialId, OrganizationIdentities.sundhedsdatastyrelsen());
```

Comment:
I'm not sure this test makes much sense, given that we expect this to become an async call.  Perhaps we can test directly on `MinLogClient` instead.